nests.json

```diff
@@ -346,6 +346,17 @@
       "status": "stable",
       "category": "scientific",
       "readme": "https://raw.githubusercontent.com/Libraries4U/Box2D/main/README.md"
-    }
+    },
+    {
+      "name": "Skylark Uploaders",
```

Comment:
Use name SkylarkUploaders instead. I see that we do not use space in name in this file.

Reply:
Done